LICENSE

```diff
@@ -1,7 +1,21 @@
-Copyright 2019 Filipe Laíns
+The MIT License (MIT)
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) 2020 openinput (https://github.com/openinput-fw)
```

Comment:
openinput is not a legal entity and as such can't hold any copyright. You can add your name here. And make the copyright year `2019-2020`.

Reply:
Didn't really think about it, I've been doing it in the code too, fixed.